pkg/sql/colconv/datum_to_vec_tmpl.go

```diff
@@ -20,9 +20,19 @@
 package colconv
 
 import (
+	"github.com/cockroachdb/apd/v2"
+	"github.com/cockroachdb/cockroach/pkg/col/coldataext"
```

Comment:
Some duplicates (and in most files). Also, as mentioned on slack, we probably need to add fake imports like
```
var _ apd.Decimal
```
in order for them to not be formatted away.

Reply:
Done.

Reply:
Thanks, that commit LGTM